java-persistence/src/main/java/com/whitecloak/training/inventory/model/Item.java

```diff
@@ -4,7 +4,9 @@
 
     private Long id;
     private String name;
-    private Long ownerId;
+    private Category category;
+    private User owner;
+    private boolean isDeleted;
```

Comment:
Deleted state is a database thing.

It should not reach the model.
Models will not exist in the first place if it was deleted.